File: loianeTraining/javaBasico/exercicios-aulas-14-15/Ex25.java
import java.util.Scanner;

public class Ex25 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.println("Responda o questionario: \"y\" - Yes | \"n\" - No");
        var score = 0;
        var response = "";

        System.out.println("Telefonou para vitima?");
        response = scan.next();
        if (response.equalsIgnoreCase("y")) { score++; }

        System.out.println("Esteve no local do crime?");
        response = scan.next();
        if (response.equalsIgnoreCase("y")) { score++; }

        System.out.println("Mora perto da vitima?");
        response = scan.next();
        if (response.equalsIgnoreCase("y")) { score++; }

        System.out.println("Devia para vitima?");
        response = scan.next();
        if (response.equalsIgnoreCase("y")) { score++; }

        System.out.println("Ja trabalhou com a vitima?");
        response = scan.next();
        if (response.equalsIgnoreCase("y")) { score++; }

        var result = "";
        if (score == 2) {
            result = "Suspeita";
        } else if (score == 3 || score == 4) {
            result = "Cumplice";
        } else if (score == 5) {
            result = "Assasino";
        } else {
            result = "Inocente";
        }

        System.out.println("Voce e " + result);
        scan.close();
    }
}

File: loiane-training-csharp/procedural-poo/c/Ex25.cs
using System;

class Ex25
{
    static void Main(string[] args)
    {
        Console.WriteLine("Responda o formulario, [1] para Sim e [0] para Nao");
        int sum = 0;

        Console.WriteLine("Telefonou para a vitima?");
        sum += int.Parse(Console.ReadLine());

        Console.WriteLine("Esteve no local do crime?");
        sum += int.Parse(Console.ReadLine());

        Console.WriteLine("Mora perto da vitima?");
        sum += int.Parse(Console.ReadLine());

        Console.WriteLine("Devia para a vitima?");
        sum += int.Parse(Console.ReadLine());

        Console.WriteLine("Ja trabalhou com a vitima?");
        sum += int.Parse(Console.ReadLine());

        if (sum == 2)
        {
            Console.WriteLine("Suspeita");
        }
        else if (sum == 3 || sum == 4)
        {
            Console.WriteLine("Cumplice");
        }
        else if (sum == 5)
        {
            Console.WriteLine("Assassino");
        }
        else
        {
            Console.WriteLine("Inocente");
        }
    }
}